mw/language.d.ts

```diff
@@ -123,7 +143,9 @@ declare global {
              * @return {Object}
              * @see https://doc.wikimedia.org/mediawiki-core/master/js/#!/api/mw.language-method-flipTransform
              */
-            function flipTransform(...tables: Array<Record<string, any>>): Record<string, any>;
+            function flipTransform<T extends Record<PropertyKey, PropertyKey>>(
+                ...tables: T[]
+            ): FlipObject<T>;
```

Comment:
This function flips keys and values of an object, I did not find a way to simplify the type of `FlipObject<T>` more. This implementation is far from being perfect, since `flipTransform( { x: 1, y: 2 } )` has type `{ number: "x" | "y" }` instead of `{ 1: "x" | 2: "y" }`.